components/org.wso2.identity.event.websubhub.publisher/src/main/java/org/wso2/identity/event/websubhub/publisher/constant/WebSubHubAdapterConstants.java

```diff
@@ -98,29 +119,44 @@ public enum ErrorMessages {
                 "Server error encountered while preparing SSL context for WebSubHub http client."),
         ERROR_CREATING_ASYNC_HTTP_CLIENT("65008", "Error while creating the Async HTTP client.",
                 "Server error encountered while creating the Async HTTP Client of WebSub Hub Adapter."),
-        TOPIC_DEREGISTRATION_FAILURE_ACTIVE_SUBS("65009", "Error occurred while de-registering topic", "Backend error" +
-                " received from WebSubHub while attempting to de-register topic: %s. Active subscribers: %s.");
+        TOPIC_DEREGISTRATION_FAILURE_ACTIVE_SUBS("65009", "Error occurred while de-registering topic ",
+                "WebSubHub returned" +
+                " an error while attempting to de-register topic: %s. Active subscribers: %s."),
+        ERROR_SUBSCRIBING_TO_TOPIC("65010", "Error subscribing to WebSub Hub topic.",
+                "Server error encountered while subscribing to the WebSub Hub topic: %s, tenant: %s, " +
+                        "callback: %s."),
+        ERROR_UNSUBSCRIBING_FROM_TOPIC("65011", "Error unsubscribing from WebSub Hub topic.",
+                "Server error encountered while unsubscribing from the WebSub Hub topic: %s, tenant: %s, " +
+                        "callback: %s."),
```

Comment:
Here we still have 'WebSub Hub'

Reply:
Fixed with https://github.com/wso2-extensions/identity-event-publishers/pull/13/commits/f58d41abd482a0452e76dba891e26e4d1fa6d915